ftplugin/spicy.vim

```diff
@@ -1 +1 @@
-set commentstring=#%s
```

Comment:
I didn't know about `comments`, that sounds like useful functionality.

Could you leave in `commentstring` though since it solve a slightly different use case? I personally use it to automatically uncomment/comment out full lines in neovim via [`gcc`](https://neovim.io/doc/user/various.html#commenting) (there is also https://github.com/tpope/vim-commentary for vim).

```suggestion
set commentstring=#%s
set comments=b:#,b:##
```